### PR TITLE
Add doc list categories

### PR DESCRIPTION
**Pull request by at-hieule**
## Description
- Doc for list categories.
## Reference Requirements
- Product Backlog ID: PBI-34
- Task ID: #34.1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Review by at-minhdao — COMMENTED** [review]
LGTM!

File: docs/api-references/categories.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```
+Get list categories
+
+#### Response - Success
+| Field | Type | Description |
+|-------|------|-------------|
+| meta | Object | object meta |
+| status | String | Status result |
+| code | Number | HTTP status codes |
+| data | Object | Object book |
```
Comment (at-duyho) [inline docs/api-references/categories.md]: - data is an array not object.

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**